xAPI-About.md

```diff
@@ -312,157 +333,181 @@ IRIs always include a scheme. This is not a requirement of this standard, but pa
+<a name="def-registration" ></a>
 
 __Registration__: An instance of a learner experiencing a particular Activity.
 
-<a name="def-rest" />
+<a name="def-rest" ></a>
 
 __Representational State Transfer (REST)__: An architecture for designing networked web Services.
 It relies on HTTP methods and uses current web best practices.
 
-<a name="def-service" />
+<a name="def-service" ></a>
 
 __Service__: A software component responsible for one or more aspects of the distributed learning process. 
 An LMS typically combines many services to design a complete learning experience.
 
-<a name="def-statement" />
+<a name="def-statement" ></a>
 
 __Statement__: A data sctructure showing evidence for any sort of experience or event which is to be tracked 
 in xAPI as a Learning Record.  A set of several Statements, each representing an event in time, might be used 
 to track complete details about a learning experience.  
 
-<a name="def-tcapi"/>
+<a name="def-tcapi"></a>
 
 __Tin Can API (TCAPI)__: The previous name of the API defined in this document, often used in 
 informal references to the Experience API.  
 
-<a name="def-verb" />
+<a name="def-verb" ></a>
 
 __Verb__: Is the action being done by the Actor within the Activity within a Statement. 
 A Verb represents the "did" in "I did this".
 
-<a name="xapi-components" />
+<a name="xapi-components" ></a>
 
 ## 5.0 xAPI Components
 
 This section explains and shows graphically how different pieces of xAPI can fit together.  
 
-##### Tracking Experiences Through Systems
+##### Tracking Experiences
 
 ![Data Flow in xAPI](/xAPIDataFlow.jpg)
 
-Figure 1 shows the tracking of experiences through the data flow of xAPI. While a learner experiences a learning activity, data about that experience is tracked to a Learning Record Store (LRS). Authentication with the LRS may use the learner's credentials, the activity provider's credentials, or a combination. A Data Analysis Tool, acting as a Service with permission, can request the data from the LRS and use it to display meaningful reports or visualizations. In the case of more structured learning, a user may be dependent on an Activity Provider to deliver and/or authorize use of content to them. Once this triangle of authorization and access is complete, a registration exists for that user in that content tracked by that LRS and experience tracking may proceed.
+Figure 1 shows the tracking of learning experiences. A learner has a Learning Experience. This experience could take place in an 
+online course, it could be on the job or it could be part of recreation. It really could be anything. This experience is tracked, 
+on the learner’s behalf, by a trusted Learning Record Provider. The Learning Record Provider may also be responsible for the 
+trusted relationship between the experience and the learner, and perhaps even launching “content” for the learner and digital 
+rights associated with the content…but it doesn’t have to be.  
```

Comment:
I'm not keen on the use of ... Can we reword this without losing the general meaning?